spec/support/features/admin_user_helpers.rb

```diff
@@ -0,0 +1,19 @@
+module Features
+  module AdminUserHelpers
+    def sign_in
+      visit '/admin'
+      fill_in 'admin_user_email', with: 'test@admin.com'
```

Comment:
Do not hardcode email and password in the method.
You should pass them as parameters.